### PR TITLE
WIP: DBZ-1068 Support capture instances with captured_column_list defined

### PR DESCRIPTION
**Pull request by grzegorz8**
https://issues.jboss.org/browse/DBZ-1068

Depends on: https://github.com/debezium/debezium-incubator/pull/60

### CODE REVIEW

**Comment by grzegorz8** [comment]
This is a very quick implementation of what we discussed in https://issues.jboss.org/browse/DBZ-1068 .

File: debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java [hunk 1]
```diff
@@ -477,6 +478,46 @@ public void blacklistColumn() throws Exception {
         stopConnector();
     }
 
+    @Test
```
Comment (grzegorz8) [inline debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java]: `@FixFor("DBZ-1068")` :)

**Comment by gunnarmorling** [comment]
@grzegorz8 Can you close this PR and open a new one against the main repository? In the course of moving the SQL Server connector out of "Incubating" state, we've moved the connector over there. Thanks!

**Comment by grzegorz8** [comment]
PR migrated

**Pull request closed without merging**